src/extension.ts

```diff
@@ -15,54 +15,9 @@ import {Mutex} from 'await-semaphore';
 const CHAR_LIMIT = 100000;
 const MAX_NUM_RESULTS = 5;
 const DEFAULT_DETAIL = "TabNine";
-const COMMAND_NAME = "TabNine Substitute";
 
 export function activate(context: vscode.ExtensionContext) {
 
-  vscode.commands.registerTextEditorCommand(
```

Comment:
this command was removed,  [SnippetString](https://code.visualstudio.com/api/references/vscode-api#SnippetString) was used instead